src/dapp/src/features/vote-creation/RoundInfo.tsx

```diff
@@ -71,7 +164,7 @@ export default function RoundInfo() {
                 field: 'snapshotFile',
                 hint: 'Upload snapshot .csv file',
                 longHint:
-                  'Upload a CSV file with a single column containing the addresses for the allowlist. Do not include a header line.',
+                  'Vote type = NO_SNAPSHOT: Upload empty file.\nVote type = NO_WEIGHTING: Upload a CSV file with a single column containing the addresses for the allowlist. Do not include a header line.\nOtherwise: Upload a CSV with 2 columns: address, weighting containing the addresses for the allowlist and their weighting. Do not include a header line.',
```

Comment:
I think we should force them to have a header even in the case that it's a single column of addresses. Just to make the parsing consistent